api/types.go

```diff
@@ -68,62 +71,62 @@ type paginator[T any] struct {
 	Token interface{} `json:"next,omitempty"`
 }
 
-type Block struct {
-	BaseFeePerGas    *hexutil.Big     `json:"baseFeePerGas,omitempty"`
-	Difficulty       hexutil.Uint64   `json:"difficulty"`
-	ExtraData        hexutil.Bytes    `json:"extraData"`
-	GasLimit         hexutil.Uint64   `json:"gasLimit"`
-	GasUsed          hexutil.Uint64   `json:"gasUsed"`
-	Hash             common.Hash      `json:"hash"`
-	LogsBloom        hexutil.Bytes    `json:"logsBloom"`
-	Miner            common.Address   `json:"miner"`
-	MixHash          common.Hash      `json:"mixHash"`
-	Nonce            types.BlockNonce `json:"nonce"`
-	Number           hexutil.Uint64   `json:"number"`
-	ParentHash       common.Hash      `json:"parentHash"`
-	ReceiptsRoot     common.Hash      `json:"receiptsRoot"`
-	Sha3Uncles       common.Hash      `json:"sha3Uncles"`
-	Size             hexutil.Uint64   `json:"size"`
-	StateRoot        common.Hash      `json:"stateRoot"`
-	Timestamp        hexutil.Uint64   `json:"timeStamp"`
-	TotalDifficulty  *hexutil.Big     `json:"totalDifficulty"`
-	Transactions     interface{}      `json:"transactiions"`
-	TransactionsRoot common.Hash      `json:"transactionsRoot"`
-	Uncles           []common.Hash    `json:"uncles"`
-}
+// type Block struct {
```

Comment:
Let's remove this if we're not using it.

Reply:
Done.